Mathlib/AlgebraicTopology/SimplicialSet.lean

```diff
@@ -351,53 +351,79 @@ lemma Truncated.hom_ext {n : ℕ} {X Y : Truncated n} {f g : X ⟶ Y} (w : ∀ n
 
 /-- The truncation functor on simplicial sets. -/
 def truncation (n : ℕ) : SSet ⥤ SSet.Truncated n :=
-  (whiskeringLeft _ _ _).obj (SimplexCategory.Δ.ι n).op
+  (whiskeringLeft _ _ _).obj SimplexCategory.Truncated.inclusion.op
 
 instance {n} : Inhabited (SSet.Truncated n) :=
   ⟨(truncation n).obj <| Δ[0]⟩
 
-section adjunctions
+
 open SimplexCategory
 
+noncomputable section
+
+/-- The n-skeleton as an endofunctor on `SSet`. -/
+abbrev sk (n : ℕ) : SSet ⥤ SSet := truncation.{u} n ⋙ lan (Truncated.inclusion.op)
```

Comment:
```suggestion
abbrev sk (n : ℕ) : SSet ⥤ SSet := truncation.{u} n ⋙ Truncated.sk n
```

Reply:
Good idea. Just swapped the order of the definitions and implemented this.